monolithic/monolithic-layered/src/main/java/com/epam/training/microservices/monolithic/model/pharmacy/Pharmacy.java

```diff
@@ -24,6 +25,7 @@ public class Pharmacy {
   private String name;
   private String address;
 
+  @ToString.Exclude
   @OneToMany(mappedBy = "pharmacy", cascade = CascadeType.ALL)
   private Set<PharmacyAvailable> available = Sets.newHashSet();
```

Comment:
Another issue discovered thanks to tests: `StackOverflowError` in `toString()` method (generated by lombok) due to cyclic dependencies between `Pharmacy` and `PharmacyAvailable` entities.

Reply:
Just for consistency, would it be better to fix the issue by adding `@ToString` annotation to the class? The reason to do that is to have all the lombok annotations in one place and fields are annotated with JPA annotations only.

Reply:
ok